package.json

```diff
@@ -598,6 +598,7 @@
     "w3c-xmlserializer": "^3.0.0",
     "wcwidth": "^1.0.1",
     "webidl-conversions": "^7.0.0",
+    "webpack-dev-server": "^4.9.1",
```

Comment:
- [ ] I ran into an error when trying to start the project. I think the reason is that you don't have a start command in your package.json file.

Error Screenshot |
:----------------------:|
![a1](https://user-images.githubusercontent.com/39523312/171997476-3ca2d7a2-8f60-4b9b-bc95-d6dd92e54ef6.png)


Below is a snippet of what your package.json script should look like:
```js
"scripts": {
  "build": "webpack",
  "start": "webpack serve --open"
}
```

Reply:
- [ ] Please implement [this](https://github.com/Abdo9826/To-Do-list/pull/3#discussion_r889515189) change about importing a file with methods related to updating the task status.

Reply:
![Screenshot (7)](https://user-images.githubusercontent.com/102296263/171998069-5b2e0a8a-cb3d-463c-bcf7-64fb5e7ee7c9.png)
 nothing changes Im working in tthis issue for a couple of days , I did every thing with no mistake

Reply:
Try deleting your node_modules and reinstalling the packages.

Reply:
I tried evry thing
i delet it and instal again no defernce

Reply:
can u just let me pass this week and I will see this bug later

Reply:
@mmsesay , I'm the next reviewer and I kindly suggest that you run `npm run build`, then open the `index.html` file in the dist folder with the live server because it is not a requirement of the project to make sure `npm run start` is working.